packages/babel-plugin-syntax-typescript/src/index.js

```diff
@@ -29,11 +29,7 @@ export default declare((api, { isTSX }) => {
       // in TS depends on the extensions, and is purely dependent on 'isTSX'.
       removePlugin(plugins, "jsx");
 
-      parserOpts.plugins.push(
-        "typescript",
-        "objectRestSpread",
-        "classProperties",
-      );
+      parserOpts.plugins.push("typescript", "classProperties");
```

Comment:
This is a breaking change if someone uses an older Babylon version. Since we are in RC, I can revert it if you prefer.